special-pages/pages/new-tab/app/components/DismissButton.jsx

```diff
@@ -0,0 +1,21 @@
+import { h } from 'preact';
+import cn from 'classnames';
+import { Cross } from './Icons';
+import { useTypedTranslation } from '../types';
+import styles from './DismissButton.module.css';
+
+const dismissText = (_t) => _t('ntp_dismiss');
```

Comment:
I think this will lose the type-safety, is there a reason to not call `t('ntp_dismiss')` directly in the component?